examples/tokenvm/actions/modify_asset.go

```diff
@@ -56,7 +56,7 @@ func (m *ModifyAsset) Execute(
 	if len(m.Metadata) > MaxMetadataSize {
 		return &chain.Result{Success: false, Units: unitsUsed, Output: OutputMetadataTooLarge}, nil
 	}
-	exists, _, supply, owner, isWarp, err := storage.GetAsset(ctx, db, m.Asset)
+	exists, _, supply, maxSupply, owner, isWarp, err := storage.GetAsset(ctx, db, m.Asset)
```

Comment:
Do you want to allow `maxSupply` to be modified here? I would think if metadata can be, then it should be?